velox/dwio/parquet/writer/arrow/Statistics.cpp

```diff
@@ -742,6 +779,55 @@ class TypedStatisticsImpl : public TypedStatistics<DType> {
     return s;
   }
 
+  std::string MinValue() const override {
+    if constexpr (std::is_same_v<T, int64_t>) {
+      if (descr_->logical_type()->is_decimal()) {
+        return encodeDecimalToBigEndian(min_);
+      }
+    }
+    if constexpr (std::is_same_v<T, int128_t>) {
+      return encodeDecimalToBigEndian(min_);
+    }
+    if constexpr (std::is_same_v<T, ByteArray>) {
+      // TODO: 16 is default value. See DEFAULT_WRITE_METRICS_MODE_DEFAULT in
+      // org.apache.iceberg.TableProperties. Need to support this table
+      // property.
+      const auto truncatedMin = UnicodeUtil::truncateStringMin(
```

Comment:
Please return std::string_view here, otherwise, may copy and cause performance issue